server/src/main/java/io/littlehorse/server/monitoring/metrics/ServerFilterRules.java

```diff
@@ -19,9 +19,9 @@ private ServerFilterRules() {}
             accept("kafka_producer_request"),
             accept("kafka_producer_outgoing_byte"),
             accept("kafka_producer_record_error"),
-            deny("kafka_stream_state"),
+            // deny("kafka_stream_state"),
```

Comment:
Hmm. Too many metrics will hurt LH Server performance. Can we hide them for a bit until we are able to configure them with the `LHServerConfig`?

Reply:
We should have a new config:
```
LHS_STREAMS_METRICS_RECORDING_LEVEL:
```